osgar/drivers/kloubak.py

```diff
@@ -22,12 +22,17 @@
 AD_CENTER = 419.7 # K2, can be modified by config
 AD_CALIBRATION_DEG = 45  # K2, can be modified by config
 AD_RANGE = -182.5  # K2, can be modified by config
+MAX_JOIN_ANGLE = 80 * math.pi/180 # K2, can be modified by config
+AD_CENTER2 = None
+AD_RANGE2 = None
 
 CAN_ID_BUTTONS = 0x1
 CAN_ID_VESC_FRONT_R = 0x91
 CAN_ID_VESC_FRONT_L = 0x92
 CAN_ID_VESC_REAR_R = 0x93
 CAN_ID_VESC_REAR_L = 0x94
+CAN_ID_VESC_BACK_R = 0x95
```

Comment:
REAR and BACK are the same thing (I know created by TomasP) - maybe reasonable would be REAR_K3_L??

Reply:
> what is the intention of config/kloubak3-camera-lidar-estop-lora.json configuration? SubT?

The intention is working osgar.go with cameras and lidars. Unfortunate, the osgar.go is not working with standart subt config.

Reply:
Ok, REAR_K3_L looks nice